### ISSUE TITLE
The 'Quit' button seems to not properly close grabber

### ISSUE THREAD

**Issue by MrAndre96** [Bionus/imgbrd-grabber#1401]
**Bug description**

When you use the 'Quit' button which can be found under `File > Quit` it will not properly close grabber. When you use this, the next time you open grabber you will get the message that grabber was not properly closed last time.

**Expected behavior**

Close it properly so it will not give a warning the next time you open grabber

**System information**

Windows 10 with the latest nightly version

**Comment by Bionus** [comment]
Indeed, it would seem that the behavior of this button is equivalent to force-quitting (kill) Grabber with the Windows Task Manager 🤔
